.github/workflows/main.yaml

```
@@ -185,6 +187,25 @@ jobs:
 
+  check-logging-standards:
+    name: Check Logging Standards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check for prohibited logging methods
+        run: |
+          echo "Checking for prohibited logging methods..."
+          GREP_RESULTS=$(grep -r --include="*.dart" -E "(debugPrint\(| log\()" lib || true)
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)